pkg/services/accesscontrol/accesscontrol.go

```diff
@@ -97,13 +97,13 @@ func BuildPermissionsMap(permissions []*Permission) map[string]bool {
 }
 
 // GroupScopesByAction will group scopes on action
-func GroupScopesByAction(permissions []*Permission) map[string]map[string]struct{} {
-	m := make(map[string]map[string]struct{})
+func GroupScopesByAction(permissions []*Permission) map[string][]string {
```

Comment:
If we need to only store unique scopes for each action we can create a "set" internally and then covert to string slice. Exposing map[string]struct{} feels a bit strange

Reply:
Yes, but i think having map[int]map[string]map[string]struct{} for Permissions on the SingedInUser is unclear what it is :)